common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -65,7 +65,7 @@ object Work {
     version: Int,
     data: WorkData[State#WorkDataState],
     state: State,
-    deletedReason: Option[DeletedReason] = None,
+    deletedReason: DeletedReason,
```

Comment:
I think this'll upset the api wrt the existing index fyi

Reply:
Yeah, I'm freezing the current pipeline.